packages/forms-web-app/src/controllers/projects/representations.js

```diff
@@ -18,6 +21,7 @@ exports.getRepresentations = async (req, res) => {
       representations: representationsResponse.data.representations,
       paginationData,
       pageOptions,
+      searchTerm,
```

Comment:
Need a `queryUrl` containing the searchTerm to be passed down for use in the pagination bar.  See documents as a reference example.

Reply:
It was already added, jest need to refresh as it was in 2nd commit

Reply:
I think you're missing the corresponding `{{queryUrl}}` being appended to the URL used within the pagination bar, i.e. `pageLinkTemplate`?

Reply:
added to pagination now